examples/event_handling/path_editor.py

```diff
@@ -100,25 +85,29 @@ def get_ind_under_point(self, event):
 
         return ind
 
-    def button_press_callback(self, event):
+    def on_draw(self, event):
+        """Callback for draws."""
+        self.background = self.canvas.copy_from_bbox(self.ax.bbox)
+        self.ax.draw_artist(self.pathpatch)
+        self.ax.draw_artist(self.line)
+        self.canvas.blit(self.ax.bbox)
+
+    def on_button_press(self, event):
         """Callback for mouse button presses."""
-        if not self.showverts:
-            return
-        if event.inaxes is None:
-            return
-        if event.button != 1:
+        if (event.inaxes is None
+                or event.button != 1
+                or not self.showverts):
             return
         self._ind = self.get_ind_under_point(event)
 
-    def button_release_callback(self, event):
+    def on_button_release(self, event):
         """Callback for mouse button releases."""
-        if not self.showverts:
-            return
-        if event.button != 1:
+        if (event.button != 1
```

Comment:
While you‘re at it! Could you change 1 to MouseButton.LEFT?

Reply:
sure, done